app/components/post_list/post/body/content/permalink_preview/permalink_preview.tsx

```diff
@@ -15,10 +16,13 @@ import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
 import {displayUsername} from '@utils/user';
 
+import PermalinkFiles from './permalink_files';
+
 import type UserModel from '@typings/database/models/servers/user';
 import type {AvailableScreens} from '@typings/screens/navigation';
 
 const MAX_PERMALINK_PREVIEW_LINES = 4;
+const MAX_PERMALINK_HEIGHT = 506;
```

Comment:
I'm not sure this is a good way to handle this, for example for long posts you would see that it calculates the maxHeight based on the screen size https://github.com/mattermost/mattermost-mobile/blob/main/app/components/post_list/post/body/message/message.tsx#L65 and I think here we should do something similar. In fact, I want to see a permalink preview, on a long post as an example and see how does it display on screen.